src/layout/content/content.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react'
+
+export interface MainContentProps {
+  readonly children: React.ReactNode
+  readonly topBarSelector?: string
+  readonly topBarHeight?: string
+  readonly tag?: string
+}
+
+export class MainContent extends Component<MainContentProps> {
+  mainContentRef = React.createRef<HTMLDivElement>()
+
+  resizeTimeout: NodeJS.Timeout = null
+
+  setContentHeight: (element?: HTMLElement) => void = (element = this.mainContentRef.current) => {
+    const contentElement = element
+    if (!contentElement) {
+      return null
+    }
+    const topBar: HTMLElement = document.querySelector(this.props.topBarSelector)
+    const height = topBar
+      ? Math.floor(window.innerHeight - topBar.offsetHeight)
+      : window.innerHeight
+
+    contentElement.style.minHeight = `${height}px`
```

Comment:
Does `px` will be transformed to another accessible unit `rem, em...` after that?

Reply:
As much as I love relative sizing for content and spacing, here I need to have the container being at minimum the height of the device's screen, so pixel values are needed for precision :)